installfrompip.adoc

```diff
@@ -66,6 +66,12 @@ linux listed above and:
 git clone https://github.com/google/grr.git
 virtualenv GRR_NEW
 source GRR_NEW/bin/activate
+# This sets up NodeJS inside the virtualenv.
+# NodeJS is needed to compile AdminUI JS/CSS files.
+pip install nodeenv
+nodeenv -p
+deactivate; source GRR_NEW/bin/activate
```

Comment:
It's deactivated and then activated back to make use of the changes nodeenv does to the bin/activate script. Saying this, deactivation doesn't seem to be necessary, we can just call "source .../activate" again.